### ISSUE TITLE
bootstrap doesn't work with ansible 2.1

### ISSUE THREAD

**Issue by sigma** [vmware/ansible-coreos-bootstrap#5]
not sure what's going on yet

```
TASK [setup] *******************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: OSError: [Errno 2] No such file or directory
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Traceback (most recent call last):\n  File \"/home/travis/.ansible/tmp/ansible-tmp-1466015750.3-53229013577475/setup\", line 114, in <module>\n    exitcode = invoke_module(module, zipped_mod, ZIPLOADER_PARAMS)\n  File \"/home/travis/.ansible/tmp/ansible-tmp-1466015750.3-53229013577475/setup\", line 28, in invoke_module\n    p = subprocess.Popen(['PATH=/home/core/bin:$PATH python', module], env=os.environ, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE)\n  File \"/usr/lib/python2.7/subprocess.py\", line 679, in __init__\n    errread, errwrite)\n  File \"/usr/lib/python2.7/subprocess.py\", line 1249, in _execute_child\n    raise child_exception\nOSError: [Errno 2] No such file or directory\n", "module_stdout": "", "msg": "MODULE FAILURE", "parsed": false}
```

**Comment by sigma** [comment]
looks like the following is no longer valid:

```
ansible_python_interpreter="PATH=/home/core/bin:$PATH python"
```

ansible now consider the whole thing as a binary. Solution is probably to make /home/core/bin/python a wrapper that does the PATH setting (it's still needed for pip and friends to just work)

**Comment by sigma** [comment]
actually a cleaner way might be to create a sitecustomize.py module to set the PATH correctly.
That's also probably simpler

**Comment by andypost** [comment]
Yep, but that also should be backward compatible

**Comment by sigma** [comment]
@andypost sure, the new solution will work with old ansible versions

**Comment by sigma** [comment]
@andypost would you mind having a look at PR #6 to see if that works for you?

**Comment by sigma** [comment]
should be fixed in version 0.6.0
